app/src/components/market/market_buy.tsx

```diff
@@ -148,6 +149,30 @@ const MarketBuyWrapper: React.FC<Props> = (props: Props) => {
 
   const disabled = (status !== Status.Ready && status !== Status.Error) || cost.isZero()
 
+  const noteBalanceCollateral = () => {
+    return (
+      <React.Fragment key="1">
```

Comment:
Unnecessary key. I'm guessing you added it because React emitted a warning when this was used in an array of components. But that `note` prop should accept any react node (right now accepts an `any`, which it should be more specific). So the solution should be simple: pass a component with these two other components instead of an array.

Reply:
Grouped the two components into one